src/test/java/io/strimzi/kproxy/ProxyTest.java

```diff
@@ -92,10 +95,61 @@ public int port(String host, int port) {
         proxy.shutdown();
     }
 
+    @Test
+    public void shouldModifyProduceMessage() throws Exception {
+        String proxyHost = "localhost";
+        int proxyPort = 9192;
+        String proxyAddress = String.format("%s:%d", proxyHost, proxyPort);
+
+        String brokerList = startKafkaCluster();
+
+        var interceptors = List.of(
+                new ApiVersionsInterceptor(),
+                new AdvertisedListenersInterceptor(new AdvertisedListenersInterceptor.AddressMapping() {
+                    @Override
+                    public String host(String host, int port) {
+                        return proxyHost;
+                    }
+
+                    @Override
+                    public int port(String host, int port) {
+                        return proxyPort;
+                    }
+                }),
```

Comment:
Isn't there some standard address mapping we can use (at least for tests like this one?)

Reply:
Good point; it was copied from the single original test; I'll try and abstract it.

Reply:
Added `FixedAddressMapping` for usage in this test.